lib/atom-scalariform.js

```diff
@@ -5,6 +5,19 @@ var packagePath = function() {
   return atom.packages.getLoadedPackage('atom-scalariform').path;
 }
 
+var projectPath = function(fileToFormatPath) {
+  var ref = atom.project.getDirectories();
+  var projectPath = '';
+  for (var i = 0, len = ref.length; i < len; i++) {
+    var directory = ref[i];
+    if (fileToFormatPath.indexOf(directory.path) > -1) {
```

Comment:
What about:

- Project: /home/user/myProject
- File: /home/user/myProject/abc/def/Main.scala
- File: /home/user/myProject/abc/Test.scala

When formatting Main.scala the `fileToFormatPath` would be /home/user/myProject/abc/def/Main.scala correct? That means if you it checked if the index of  /home/user/myProject/abc/ was > 1 it would return true as that index would actually be 0?